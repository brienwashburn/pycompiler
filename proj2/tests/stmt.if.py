if a: pass
elif b: pass
elif c: pass
elif d: pass
else: pass

if a: pass

if a:
    pass

if a and b:
    pass

if a and b:
    pass
elif b and c:
    pass

if a and b:
    pass
else:
    c
