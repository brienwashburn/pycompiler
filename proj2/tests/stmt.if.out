(Module
 (If
  (test (Name a))
  (body (Pass))
  (orelse
   (If
    (test (Name b))
    (body (Pass))
    (orelse
     (If
      (test (Name c))
      (body (Pass))
      (orelse (If (test (Name d)) (body (Pass)) (orelse (Pass)))))))))
 (If (test (Name a)) (body (Pass)) (orelse))
 (If (test (Name a)) (body (Pass)) (orelse))
 (If (test (BoolOp And (Name a) (Name b))) (body (Pass)) (orelse))
 (If
  (test (BoolOp And (Name a) (Name b)))
  (body (Pass))
  (orelse (If (test (BoolOp And (Name b) (Name c))) (body (Pass)) (orelse))))
 (If
  (test (BoolOp And (Name a) (Name b)))
  (body (Pass))
  (orelse (Expr (Name c)))))
